website/content/api-docs/system/internal-counters.mdx

```diff
@@ -1163,6 +1166,11 @@ $ curl \
 }
 ```
 
+<Note>
+    As of 1.15.13, 1.16.7, 1.17.3, 1.18 and later, the billing start date (license start date if not configured) will automatically rollover to the latest billing year. 
+    Refer to <a href="/vault/docs/upgrading/upgrade-to-1.15.x#auto-roll-billing-start-date">Upgrade to 1.15.x</a>, <a href="/vault/docs/upgrading/upgrade-to-1.16.x#auto-roll-billing-start-date">Upgrade to 1.16.x</a>, <a href="/vault/docs/upgrading/upgrade-to-1.17.x#auto-roll-billing-start-date">Upgrade to 1.17.x</a>, <a href="/vault/docs/upgrading/upgrade-to-1.18.x#auto-roll-billing-start-date">Upgrade to 1.18.x</a>, for additional information on auto-roll billing start date.
```

Comment:
```suggestion
<Note title="Important change to supported versions">

    Billing start date, or license start date if you do not have an explicit
    billing start date configured, automatically rolls over to the latest
    billing year at the end of the last cycle.

    For more information, refer to the upgrade guide for your Vault version:
    
    - [Vault v1.16.x](/vault/docs/upgrading/upgrade-to-1.16.x#auto-roll-billing-start-date)
    - [Vault v1.17.x](/vault/docs/upgrading/upgrade-to-1.17.x#auto-roll-billing-start-date)
    - [Vault v1.18.x](/vault/docs/upgrading/upgrade-to-1.18.x#auto-roll-billing-start-date)

</Note>
```

Technically, 1.15 won't be a supported version when 1.18 releases, and we don't typically call out unsupported versions in the docs. If it's live now for 1.15.x, we should update the relevant 1.15 docs in a separate PR since these docs won't publish until 1.18 RC (at which point, we won't update 1.15 docs regularly)

Also, just as an FYI, you don't need to use HTML in the asides if you include whitespace before/after the body of the text. It's a bug in our content parser that it expects anything within a tag structure to be HTML unless there's whitespace to sort of "reset" the parser's  logic 🤷🏽

Reply:
I think we will be releasing the changing in 1.16.x and 1.17.x. I will remove the 1.15 changes from this PR

Reply:
I tried checking this on the website and the links don't show up unless we use <a href>. The links above did not work and shows up as text.